OpenRA.Mods.Common/Projectiles/Missile.cs

```diff
@@ -849,9 +848,8 @@ public void Tick(World world)
 
 			// Check for walls or other blocking obstacles
 			var shouldExplode = false;
-			WPos blockedPos;
 			if (info.Blockable && BlocksProjectiles.AnyBlockingActorsBetween(world, lastPos, pos, info.Width,
-				out blockedPos))
+				    out var blockedPos))
```

Comment:
Spaces.

Reply:
Also imo we should move that onto the line above if we can. (Line length limit?)